src/svgcontext.ts

```diff
@@ -60,13 +116,11 @@ export class SVGContext implements RenderContext {
   parent: SVGGElement;
   groups: SVGGElement[];
   fontString: string = '';
-  fontSize: number = 0;
-  ie: boolean = false; // true if the browser is Internet Explorer.
 
   constructor(element: HTMLElement) {
     this.element = element;
 
-    const svg = this.create('svg') as SVGSVGElement;
+    const svg = this.create('svg');
```

Comment:
Nice work with the overload signatures. Makes this a bit cleaner.

Reply:
Yeah, TypeScript's type system is pretty nice on the whole.

Reply:
+1